operator/apis/loki/v1/lokistack_types.go

```diff
@@ -992,6 +992,8 @@ const (
 	ReasonQueryTimeoutInvalid LokiStackConditionReason = "ReasonQueryTimeoutInvalid"
 	// ReasonNoZoneAwareNodes when the cluster does not contain any nodes with the labels needed for zone-awareness.
 	ReasonNoZoneAwareNodes LokiStackConditionReason = "ReasonNoZoneAwareNodes"
+	// ReasonTopologyKeyNodeLabelsEmpty when the topologyKey node label has an empty value
+	ReasonTopologyKeyNodeLabelsEmpty LokiStackConditionReason = "ReasonTopologyKeyNodeLabelsEmpty"
```

Comment:
Now that we have two degraded conditions regarding zone-awareness, maybe it does make sense to "group them". Maybe something like this?

```suggestion
	// ReasonZoneAwareNodesMissing when the cluster does not contain any nodes with the labels needed for zone-awareness.
	ReasonZoneAwareNodesMissing LokiStackConditionReason = "ReasonZoneAwareNodesMissing"
	// ReasonZoneAwareEmptyLabel when the node-label used for zone-awareness has an empty value.
	ReasonZoneAwareEmptyLabel LokiStackConditionReason = "ReasonZoneAwareEmptyLabel"
```